### ISSUE TITLE
Logits Shift Problem

### ISSUE THREAD

**Issue by guoyd1999** [Gen-Verse/dLLM-RL#43]
Do trado models shift their logits during the SFT/RL process? I checked their SFT forward function, and it looks the same as the SDAR model’s. So they do not shift their logits to predict the next token, right? Also, in the sampling process, it does not shift its prediction. Any information that I missed?
